modules/openideal_faq/openideal_faq.module

```diff
@@ -5,9 +5,32 @@
  * Primary module hooks for openideal_faq module.
  */
 
+use Drupal\Core\Form\FormStateInterface;
+
 /**
  * Implements hook_multiple_field_remove_button_field_types_alter().
  */
 function openideal_faq_multiple_field_remove_button_field_types_alter(array &$fieldTypes) {
   $fieldTypes[] = 'faqfield';
 }
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ */
+function openideal_faq_field_widget_faqfield_default_form_alter(&$element, FormStateInterface $form_state, $context) {
+  $element['answer']['#allowed_formats'] = ['full_html'];
+  $element['#after_build'][] = '_remove_textarea_help';
```

Comment:
Can't we use `_allowed_formats_remove_textarea_help()` to avoid duplicated code?

Reply:
I have updated the commit, please check.

Reply:
Looks much better, thanks!